azext_iot/central/commands_device_template.py

```diff
@@ -16,23 +17,25 @@ def get_device_template(
     app_id: str,
     device_template_id: str,
     token=None,
-    central_dns_suffix="azureiotcentral.com",
+    central_dns_suffix=CENTRAL_ENDPOINT,
 ):
     provider = CentralDeviceTemplateProvider(cmd=cmd, app_id=app_id, token=token)
-    return provider.get_device_template(
+    template = provider.get_device_template(
         device_template_id=device_template_id, central_dns_suffix=central_dns_suffix
     )
+    return template.template
```

Comment:
_super nit_: Is there a different identifier you can use for the payload returned from `get_device_template` (like `device_template`) so you can avoid `template.template`?

Reply:
I have some follow up work items, I'll tackle this as part of that

Reply:
#wontfix